### PR TITLE
Unit converter

### PR DESCRIPTION
**Pull request by abdallahsow**

### CODE REVIEW

**Comment by abdallahsow** [comment]
[https://www.figma.com/file/JxMP4NYqNNtvNdljBnyrja/Unit-Conversion-(Copy)?node-id=0%3A13](url)

File: Unit-converter/style.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 1350px;
+  background-color: #1c1c1c;
+  margin: 50px auto;
+  padding: 120px 70px;
+  display: flex;
+  gap: 70px;
```
Comment (ahamad-ch) [inline Unit-converter/style.css:13]: - [ ] apply `justify-content: center;`

**Review by ahamad-ch — CHANGES_REQUESTED** [review]
- [ ] Make sure to apply a maxlength for this input field and let the input field have a fixed width as its overflowing
image

**Review by ahamad-ch — CHANGES_REQUESTED** [review]
- [ ] Fix the merge conflicts

**Pull request merged**